### ISSUE TITLE
When Git Bash is used as the terminal, download progress isn't updated until completion.

### ISSUE THREAD

**Issue by kittenparry** [kittenparry/meituri-downloader#1]
In Windows Command Prompt you get progress of each image download as they are happening like below:
```
Downloading http://url/image.jpg.
Complete. Took 3.52 seconds.
```
But on Git Bash it doesn't happen until completion, creating an illusion of a halted/suspended or a not-working program. User can still check the download folder to see images appearing, doesn't help when the person doesn't realise the fact, though.

![screenshot_git_bash_1](https://user-images.githubusercontent.com/12991847/52343940-921edd00-2a2a-11e9-922a-a2423a316d5a.png)
![screenshot_git_bash_2](https://user-images.githubusercontent.com/12991847/52343951-93e8a080-2a2a-11e9-98f2-1f7ca8e027fe.png)

**In short:** Stays the way it is in the first image until all downloads are complete.  
Expected: Should print out each line as images are downloaded one by one.

**Comment by kittenparry** [comment]
Could be a problem with other terminals as well.

**Comment by kittenparry** [comment]
Doesn't seem to be a problem in `urxvt` in Manjaro. Maybe it was isolated to Git Bash in Windows.
